Sources/SwiftProtobuf/TextFormatEncodingVisitor.swift

```diff
@@ -96,13 +98,17 @@ internal struct TextFormatEncodingVisitor: Visitor {
               var bytes = Internal.emptyData
               try decoder.decodeSingularBytesField(value: &bytes)
               bytes.withUnsafeBytes { (p: UnsafePointer<UInt8>) -> () in
-                  var testDecoder = BinaryDecoder(forReadingFrom: p, count: bytes.count)
+                  var testDecoder = BinaryDecoder(forReadingFrom: p,
+                                                  count: bytes.count,
+                                                  parent: decoder)
```

Comment:
Should we plan to add recursion depth limits to the TextFormat decoder as well?  If so, would it use a separate `Options` struct?